packages/procaptcha-react/README.md

```diff
@@ -19,7 +19,7 @@ See the [client example](https://github.com/prosopo/client-example) for a minima
 in a frontend app.
 
 ```jsx
-<Procaptcha config={config} callbacks={{ onAccountNotFound, onError, onHuman, onExpired }} />
+<Procaptcha config={config} callbacks={{ onError, onHuman, onExpired }} />
```

Comment:
what happened to the callback here?

Reply:
It doesn't do anything different to the error callback, refactored so that it's now treated as a regular error

Reply:
It's removed from everywhere yep.

I was thinking about this, and I'm not sure that an account not found error is any more special than the other error types really (e.g. extension not found, no accounts found, network not found, the various no/invalid captcha errors, sitekey errors, etc)

Reply:
I concur good sir